### ISSUE TITLE
[NPC][WotlK] Sifreldar Runekeeper and Storm Maiden, missing emotes/paths/spawn locs

### ISSUE THREAD

**Issue by Teridan-dal-wow** [dalaranwow/dalaran-wow#4296]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Part 1:
Npc Sifreldar Runekeeper http://wotlk.openwow.com/npc=29331 and Sifreldar Storm Maiden http://wotlk.openwow.com/npc=29323 has missing emotes, missing or wrong paths and some missing spawn locations.
Overview of the area relevent of this report.
![wowscrnshot_012317_142710](https://cloud.githubusercontent.com/assets/6797611/22204038/9a3a7bcc-e178-11e6-9bd9-188b9f348621.jpg)
![wowscrnshot_012317_142142](https://cloud.githubusercontent.com/assets/6797611/22204041/9fbe93c6-e178-11e6-91b1-c8c1b35039b2.jpg)

Part 2 - Missing spawn locations:

![wowscrnshot_012317_145625](https://cloud.githubusercontent.com/assets/6797611/22204929/50e99ec6-e17d-11e6-9ea8-0b65b9436299.jpg)
![wowscrnshot_012317_145632](https://cloud.githubusercontent.com/assets/6797611/22204937/569095fa-e17d-11e6-8809-013a69a1cf86.jpg)
![wowscrnshot_012317_145700](https://cloud.githubusercontent.com/assets/6797611/22204942/5c1ad2c4-e17d-11e6-9136-9c638a746710.jpg)
![wowscrnshot_012317_145718](https://cloud.githubusercontent.com/assets/6797611/22204945/60a3d5b6-e17d-11e6-8bb4-ff5adcf70282.jpg)
![wowscrnshot_012317_145733](https://cloud.githubusercontent.com/assets/6797611/22204947/636064ae-e17d-11e6-832e-93f87d080d4b.jpg)

Part 3 - Wrong spawns (these are probably some of the npcs from the pictures above, but on incorrect place).

![wowscrnshot_012317_150904](https://cloud.githubusercontent.com/assets/6797611/22205184/478cde8c-e17e-11e6-8ae2-b0695f69c95a.jpg)
![wowscrnshot_012317_150921](https://cloud.githubusercontent.com/assets/6797611/22205185/4ab9e5dc-e17e-11e6-99f2-efe84d097a78.jpg)

Part 4 - Incomplete path. That path of this npc should be longer.
![wowscrnshot_012317_151335](https://cloud.githubusercontent.com/assets/6797611/22205296/cb431a5c-e17e-11e6-88b7-b1c37630498b.jpg)

 - This npc also, when engaged, it casts frostbolt while moving and it continue the path while fighting.

Part 5 - Missing cage and goblin. 
Its relevent to quest: They Took Our Men! http://wotlk.openwow.com/quest=12843
![wowscrnshot_012317_152238](https://cloud.githubusercontent.com/assets/6797611/22205514/daec10ac-e17f-11e6-9186-bc29db1bd944.jpg)

Part 6 - Missing emotes (check video below).
**Expected behaviour**: Video from retail legion from 23.01.2017:
Part 1 - https://www.youtube.com/watch?v=1Z5vr5XSEkY
Part 2 - https://www.youtube.com/watch?v=68ZoKR7lSlQ

**Comment by Rushor** [comment]
i will add everything this evening :)

**Comment by Rushor** [comment]
just aguck note here: the "incomplete path" npc was perfectly sniffed

wps for thim:

```sql
-- Pathing for  Entry: Entry: 'TDB FORMAT' 
SET @NPC := XXXXXX;
SET @PATH := @NPC * 10;
UPDATE `creature` SET `spawndist`=0,`MovementType`=2,`position_x`=6770.527,`position_y`=-1021.021,`position_z`=766.3752 WHERE `guid`=@NPC;
DELETE FROM `creature_addon` WHERE `guid`=@NPC;
INSERT INTO `creature_addon` (`guid`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES (@NPC,@PATH,0,0,1,0, '');
DELETE FROM `waypoint_data` WHERE `id`=@PATH;
INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
(@PATH,1,6770.527,-1021.021,766.3752,0,0,0,0,100,0), -- 12:41:52
(@PATH,2,6774.866,-1027.486,766.3826,0,0,0,0,100,0), -- 12:41:52
(@PATH,3,6779.206,-1033.951,766.39,0,0,0,0,100,0), -- 12:41:52
(@PATH,4,6779.206,-1033.951,766.39,0,0,0,0,100,0), -- 12:41:52
(@PATH,5,6779.391,-1033.916,766.4652,0,0,0,0,100,0), -- 12:41:53
(@PATH,6,6781.516,-1056.379,766.7101,0,0,0,0,100,0), -- 12:42:01
(@PATH,7,6779.214,-1068.532,766.6703,0,0,0,0,100,0), -- 12:42:06
(@PATH,8,6792.936,-1106.229,772.7088,0,0,0,0,100,0), -- 12:42:12
(@PATH,9,6810.756,-1122.94,785.3698,0,0,0,0,100,0), -- 12:42:24
(@PATH,10,6790.958,-1103.771,771.4297,0,0,0,0,100,0), -- 12:42:35
(@PATH,11,6791.321,-1104.281,771.5864,0,0,0,0,100,0), -- 12:42:35
(@PATH,12,6791.684,-1104.791,771.743,0,0,0,0,100,0), -- 12:42:35
(@PATH,13,6792.842,-1106.42,772.618,0,0,0,0,100,0), -- 12:42:35
(@PATH,14,6793.759,-1107.705,773.0178,0,0,0,0,100,0), -- 12:42:35
(@PATH,15,6795.971,-1109.732,774.243,0,0,0,0,100,0), -- 12:42:35
(@PATH,16,6797.445,-1111.084,774.868,0,0,0,0,100,0), -- 12:42:35
(@PATH,17,6798.183,-1111.76,775.493,0,0,0,0,100,0), -- 12:42:35
(@PATH,18,6799.657,-1113.111,776.118,0,0,0,0,100,0), -- 12:42:35
(@PATH,19,6800.395,-1113.787,776.7781,0,0,0,0,100,0), -- 12:42:35
(@PATH,20,6801.869,-1115.139,777.5281,0,0,0,0,100,0), -- 12:42:35
(@PATH,21,6802.606,-1115.814,778.4031,0,0,0,0,100,0), -- 12:42:35
(@PATH,22,6804.081,-1117.166,779.1531,0,0,0,0,100,0), -- 12:42:35
(@PATH,23,6804.818,-1117.842,780.0281,0,0,0,0,100,0), -- 12:42:35
(@PATH,24,6806.293,-1119.193,781.4031,0,0,0,0,100,0), -- 12:42:35
(@PATH,25,6807.768,-1120.545,782.2781,0,0,0,0,100,0), -- 12:42:35
(@PATH,26,6808.505,-1121.221,783.2781,0,0,0,0,100,0), -- 12:42:35
(@PATH,27,6809.979,-1122.572,784.2781,0,0,0,0,100,0), -- 12:42:35
(@PATH,28,6810.717,-1123.248,785.1531,0,0,0,0,100,0), -- 12:42:35
(@PATH,29,6812.191,-1124.6,786.1531,0,0,0,0,100,0), -- 12:42:35
(@PATH,30,6812.191,-1124.6,786.1531,0,0,0,0,100,0), -- 12:42:35
(@PATH,31,6824.273,-1140.217,796.6036,0,0,0,0,100,0), -- 12:42:36
(@PATH,32,6820.466,-1157.098,805.7188,0,0,0,0,100,0), -- 12:42:46
(@PATH,33,6805.752,-1163.74,811.014,0,0,0,0,100,0), -- 12:42:53
(@PATH,34,6818.727,-1159.172,806.4229,0,0,0,0,100,0), -- 12:43:03
(@PATH,35,6824.18,-1142.572,798.067,0,0,0,0,100,0), -- 12:43:09
(@PATH,36,6812.635,-1125.064,786.6625,0,0,0,0,100,0), -- 12:43:18
(@PATH,37,6795.604,-1109.136,773.9398,0,0,0,0,100,0), -- 12:43:26
(@PATH,38,6780.573,-1088.984,767.4163,0,0,0,0,100,0), -- 12:43:38
(@PATH,39,6775.638,-1081.745,766.4496,0,0,0,0,100,0), -- 12:43:52
(@PATH,40,6779.679,-1068.467,766.4709,0,0,0,0,100,0), -- 12:43:57
(@PATH,41,6781.518,-1056.111,766.6932,0,0,0,0,100,0), -- 12:44:01
(@PATH,42,6779.147,-1033.783,766.6435,0,0,0,0,100,0), -- 12:44:11
(@PATH,43,6774.77,-1027.126,766.6381,0,0,0,0,100,0), -- 12:44:14
(@PATH,44,6759.426,-1017.7,766.6392,0,0,0,0,100,0), -- 12:44:21
(@PATH,45,6756.741,-1000.856,767.1541,0,0,0,0,100,0), -- 12:44:25
(@PATH,46,6756.155,-1001.621,766.9041,0,0,0,0,100,0), -- 12:44:32
(@PATH,47,6753.849,-1010.317,766.4717,0,0,0,0,100,0), -- 12:44:36
(@PATH,48,6759.732,-1017.921,766.6362,0,0,0,0,100,0), -- 12:44:40
(@PATH,49,6775.08,-1027.515,766.6372,0,0,0,0,100,0), -- 12:44:46
(@PATH,50,6779.468,-1034.036,766.4645,0,0,0,0,100,0), -- 12:44:50
(@PATH,51,6781.6,-1056.437,766.7156,0,0,0,0,100,0), -- 12:45:00
(@PATH,52,6779.321,-1068.573,766.6611,0,0,0,0,100,0), -- 12:45:05
(@PATH,53,6792.881,-1106.05,772.7088,0,0,0,0,100,0), -- 12:45:10
(@PATH,54,6810.71,-1123.079,785.4041,0,0,0,0,100,0), -- 12:45:23
(@PATH,55,6824.344,-1140.323,796.4342,0,0,0,0,100,0), -- 12:45:34
(@PATH,56,6820.311,-1157.064,805.7056,0,0,0,0,100,0); -- 12:45:44
-- 0x203AF447601CA4C000279D000005E489 .go 6770.527 -1021.021 766.3752
```

**Comment by Teridan-dal-wow** [comment]
Great! Happy to hear the sniffing is helping you.

**Comment by Rushor** [comment]
DB/Creature: Sifreldar Runekeeper and Storm Maiden Add missing emotes/paths/spawn locs

* Add missing creature Spawns
* Add one missing Cage
* update position for one creature
* Add emotes
* Delete wrong spawns
* Add Missing Path
* Also add a generic way to remove UNIT_FIELD_BYTES_1 flags from creatures on oggro
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4296
* commit: 227a8e7d80d4d5146ab83d8127377ed7a6a3b89e

![GitHub Logo](http://i.imgur.com/tNm5Xwh.jpg)
![GitHub Logo](http://i.imgur.com/rXjFapF.jpg)
![GitHub Logo](http://i.imgur.com/1PiVMWi.jpg)
![GitHub Logo](http://i.imgur.com/V4C7B3u.jpg)
![GitHub Logo](http://i.imgur.com/UZM7SXP.jpg)
![GitHub Logo](http://i.imgur.com/5eMzw1J.jpg)
![GitHub Logo](http://i.imgur.com/YFdFmR1.png)
![GitHub Logo](http://i.imgur.com/5MDxvYX.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Teridan-dal-wow** [comment]
Feedback for this report from TrinityCore rev. 02633a332cf7 2017-02-09.

I will asume this was not pushed to tc:

> Also add a generic way to remove UNIT_FIELD_BYTES_1 flags from creatures on oggro

Other than that everything else works exacly like on retail. Thank you for your work!

**Comment by Rushor** [comment]
yep that generic fix is not pushed to tc. in general i'm going to push less things to tc from now on, since you will be able to test them on dalaran-ptr in near future (i no longer need tc for feedback then :D )

**Comment by Rushor** [comment]
Also add a generic way to remove UNIT_FIELD_BYTES_1 flags from creatures on oggro

- > this was fixed on PTR with commit 6150cb04e925a9992b23e656fb176eb65eae5bae
